### PR TITLE
[UX] Add a tooltip to explain what is Enclosure, Escape and Delimiter on Contacts import

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
This update adds tooltips for the terms ‘Enclosure’, ‘Escape’, and ‘Delimiter’ in the Contacts import section. Recognizing the technical nature of these concepts, the tooltips provide straightforward explanations, aiding users in troubleshooting import issues. 

This enhancement fosters a more informative and user-friendly environment, empowering users to manage their imports with greater confidence and efficiency.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Open dropdown
4. Click to import
5. Hover the mouse on the labels

image

### CODE REVIEW

**Comment by andersonjeccel** [comment]
@RCheesley 

Would you mind adding these icons to me?:
- Bug reports
- Reviewed pull requests
- User testing
- Project management
- Translation

**Review by LordRembo — COMMENTED** [review]
More tooltips certainly seems like a good idea. Not quite sure about the explanations you are using though (eg. 'signal' and 'secret code'). I think it's a bit too convoluted. The text can be simplified and more generic.  
Just some examples:
Eg. for the delimiter tooltip, it could just be: `This is the character that separates each piece of etc…`
Eg. for enclosure: `This is a character used to enclose (mark the start and end of) a piece of etc…`
Eg. for escape: `This is a character used to precede a special character, so the system doesn't get confused, often etc…

Let's see what the others think.

**Comment by code5rick** [comment]
I understood @andersonjeccel's perspective. He's trying to explain it to users who may not have programming skills, such as marketers.

> Not quite sure about the explanations you are using though (eg. 'signal' and 'secret code'). I think it's a bit too convoluted.

However, considering that we're dealing with imports and a more technical task that may exceed the knowledge of marketers, we should provide an explanation that accurately reflects the technical nature of the task. It's important to remember that we also have documentation available to assist all users.

**Comment by LordRembo** [comment]
@code5rick Would you agree with the suggested changes then, or how would you rewrite the text to better reflect the technical nature?

**Comment by code5rick** [comment]
@LordRembo I would follow your suggestion:

> Eg. for the delimiter tooltip, it could just be: This is the character that separates each piece of etc…
> Eg. for enclosure: This is a character used to enclose (mark the start and end of) a piece of etc…
> Eg. for escape: `This is a character used to precede a special character, so the system doesn't get confused, often etc…

**Comment by andersonjeccel** [comment]
I'd suggest "symbol" instead of character since it's more specific

**Comment by LordRembo** [comment]
@andersonjeccel I get what you mean but 'character' is the most common way to refer to these elements. The quote or comma are very rudimentary and common elements, just like letters. They are not usually referred to as 'symbols'.

**Comment by Mike-Dropsolid** [comment]
Great initial proposal and great feedback. You guys are on fire!

+1 for Rembrands proposal.

**Comment by RCheesley** [comment]
> @RCheesley
> 
> Would you mind adding these icons to me?:
> 
> * Bug reports
> * Reviewed pull requests
> * User testing
> * Project management
> * Translation

I don't understand the request?

I think that the proposals from @LordRembo are sound - we should be using the language that is used when importing and exporting and ensure we explain it in layman's terms.

@LordRembo perhaps you could get into the habit of using 'suggestion' when reviewing PRs which makes it easier to discuss the actual proposed changes within the context of the file, and makes it quicker for the author to implement, too 😄

**Comment by andersonjeccel** [comment]
@RCheesley On this table, IDK what's called 

image

**Comment by escopecz** [comment]
Please, pay attention to the Github Actions notifications. We cannot merge a PR that is failing something otherwise it will block merging the rest of the PRs.

**Comment by escopecz** [comment]
@andersonjeccel why is this labeled as ready to commit? I cannot find any approval.

**Comment by andersonjeccel** [comment]
@escopecz Commit of the requested changes caused the review to reset

**Comment by andersonjeccel** [comment]
@escopecz I think the failed CS test might not be caused by the PR itself

**Comment by andersonjeccel** [comment]
@escopecz now the cs issue appeared, comitted a fix

**Comment by escopecz** [comment]
If I understand correctly then all the reviewers wants to swap "symbol" with "character". I'm removing the ready to commit label.

Edit: The label was removed earlier. Let's wait for the update.

**Review by Esthertests — APPROVED** [review]
Nice
image

on prod
image

**Comment by andersonjeccel** [comment]
@RCheesley Ready-to-commit?

**Review by escopecz — APPROVED** [review]
👍
